spinalcordtoolbox/scripts/sct_extract_metric.py

```diff
@@ -369,10 +369,17 @@ def main(argv: Sequence[str]):
 
     # Combine individual labels for estimation
     if combine_labels:
-        # Add entry with internal ID value (99) which corresponds to combined labels
-        label_struc[99] = LabelStruc(id=labels_id_user, name=','.join([str(i) for i in labels_id_user]),
-                                     map_cluster=None)
-        labels_id_user = [99]
+        # Avoid case where user asked to combine a single label which is already a combination of 'native' labels
+        # Related to https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4467
+        if not (len(labels_id_user)==1 and labels_id_user[0] in combined_labels_ids):
+            # If user is trying to combine more than one label that is part of CombinedLabels, exit with error
+            if len(labels_id_user)!=1 and any(element in combined_labels_ids for element in labels_id_user):
+                printv('\nERROR: You are trying to combine multiple labels that are already combined (under section "# Combined labels" the info_label.txt file. Instead, enter the all the labels that you wish to combine from the list "# Keyword=IndivLabels".', 1, type='error')
```

Comment:
It looks like this case is supposed to handle what you said in the PR description:

> > * Labels under `Keyword=CombinedLabels`: `-l 51,52 -combine 1` --> Now that's an interesting one. I'm now sure what will happen in the code to be honest 😅 but ideally we would produce a label (99) that combines all tracts within 51 and 52 (and deal with the redundancy)

I'm not sure either, to be honest? But, if we just want to handle this error to unblock @Kaonashi22 right now, we can merge this as-is, and then I can follow up with better testing for this specific case.

Reply:
right-- my comment on github is obsolete-- I ended not _not_ solving it, and instead throw an error, and i fixed the batch processing file for @Kaonashi22 here: https://github.com/sct-pipeline/spine-park/commit/3bd95801f40b9dbfc78591ce586e8cd8bd548c51

Reply:
I didn't properly answered your comment: 

> and then I can follow up with better testing for this specific case (to allow the behavior without throwing an error

to be honest i'm not sure this is high priority-- this is the first time in 5y this feature was required, and it is pretty easy to fix (by just listing all tracts, which is really not a big deal). Maybe we can open an issue, but that doesn't need to be high priority